install-dev/theme/views/content.php

```diff
@@ -0,0 +1,124 @@
+   * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+   */
+?>
+
+<div id="contentInfosBlock">
+  <h2><?php echo $this->translator->trans('Configure content', [], 'Install'); ?></h2>
+
+  <div class="field clearfix">
+    <label class="aligned"><?php echo $this->translator->trans('Installation of theme', [], 'Install'); ?></label>
+    <div class="contentinput">
+      <div class="themes-container">
+        <?php foreach ($this->themes as $theme): ?>
+          <div class="theme-card">
+            <label>
+              <input value="<?php echo $theme->get('name'); ?>" type="radio" name="theme" style="vertical-align: middle;" <?php if ($this->session->content_theme === $theme->get('name')): ?>checked="checked"<?php endif ?> autocomplete="off" />
+              <?php echo $theme->get('display_name'); ?>
+              <div class="image-block">
+                <img src="../<?php echo $theme->get('preview'); ?>" alt="<?php echo $theme->get('display_name'); ?>">
+              </div>
+
+              <p class="theme-version">
+                <strong><?php echo $this->translator->trans('Version:', [], 'Install'); ?></strong> <?php echo $theme->get('version'); ?>
+              </p>
+            </label>
+          </div>
+        <?php endforeach; ?>
+      </div>
+    </div>
+
+    <p class="userInfos aligned"><?php echo $this->translator->trans('Select the theme to install.', [], 'Install'); ?></p>
+  </div>
+
+  <div class="field clearfix">
+    <label class="aligned"><?php echo $this->translator->trans('Installation of demo products', [], 'Install'); ?></label>
+    <div class="contentinput">
+      <label>
+        <input value="1" type="radio" name="install-fixtures" style="vertical-align: middle;" <?php if ($this->session->content_install_fixtures): ?>checked="checked"<?php endif; ?> autocomplete="off" />
+        <?php echo $this->translator->trans('Yes', [], 'Install'); ?>
+      </label>
+      <label>
+        <input value="0" type="radio" name="install-fixtures" style="vertical-align: middle;" <?php if (!$this->session->content_install_fixtures): ?>checked="checked"<?php endif; ?> autocomplete="off" />
+        <?php echo $this->translator->trans('No', [], 'Install'); ?>
+      </label>
+    </div>
+    <p class="userInfos aligned"><?php echo $this->translator->trans('Demo products are a good way to learn how to use PrestaShop. You should install them if you are not familiar with it.', [], 'Install'); ?></p>
+  </div>
+
+  <div class="field clearfix">
+    <label class="aligned"><?php echo $this->translator->trans('Installation of modules', [], 'Install'); ?></label>
+    <div class="contentinput">
+      <ul class="modules-select-type">
+        <li>
+          <label>
+            <input value="<?php echo static::MODULES_ALL; ?>" name="module-action" type="radio" autocomplete="off"<?php if ($this->moduleAction === static::MODULES_ALL): ?> checked="checked"<?php endif; ?>/>
```

Comment:
The view can access `InstallControllerHttpContent`'s consts? 🤯

Reply:
Installation views are `include` so yes, everything inside it is available :)